standard/variables.md

```diff
@@ -152,20 +152,26 @@ A local variable introduced by a *local_variable_declaration* or *declaration_ex
 
 #### §discards-new-clause Discards
 
-A ***discard*** is a local variable that has no name and can thus only be referred to once, at the point where it is introduced. A discard is not initially assigned, so it is always an error to access its value. A discard is introduced by a *declaration_expression* with the identifier `_`, and by use of the *simple_name* `_` when lookup of that name does not find a declaration ([§11.7.4](expressions.md#1174-simple-names)).
+A ***discard*** is a local variable that has no name. A discard is introduced by a declaration expression (§declaration-expressions-new-clause) with the identifier `_`; and is either implicitly typed (`_` or `var _`) or explicitly typed (`T _`).
 
-*Example:*
+> *Note*: `_` is a valid identifier in many forms of declarations. *end note*
+  
+Because a discard has no name, the only reference to the variable it represents is the expression that introduces it. A discard is not initially assigned, so it is always an error to access its value.
```

Comment:
As the example below shows a discard can be passed as an `out` parameter, so within the called method the variable *must* be referenced as the parameter name is an alias for it. I know what this is trying to say but wordsmithing a suggestion escapes me at the moment.

Reply:
I added a note; hopefully that's enough wordsmithing to get the point across.